ghost/collections/src/Collection.ts

```diff
@@ -0,0 +1,15 @@
+// @NOTE: file names having only type declarations should also
+//        be uppercased
+/* eslint-disable ghost/filenames/match-regex */
```

Comment:
@allouis I think files only including `type` export should be allowed to be capitalized too 🤔